.github/CODEOWNERS

```
@@ -237,7 +237,7 @@ packages/react-components/react-storybook-addon-export-to-sandbox @microsoft/flu
 packages/react-components/babel-preset-storybook-full-source @microsoft/fluentui-react-build
 packages/react-components/react-jsx-runtime @microsoft/teams-prg
 packages/react-components/react-toast @microsoft/teams-prg
-packages/react-components/react-search-preview @microsoft/cxe-red
+packages/react-components/react-search @microsoft/cxe-red
```

Should you also add your user name here?

```suggestion
packages/react-components/react-search @microsoft/cxe-red @smhigley
```

---

Sure, why not 😅